src/components/learner-credit-management/BudgetCard.jsx

```diff
@@ -1,168 +1,67 @@
-import React, { useState } from 'react';
```

Comment:
The original `BudgetCard.jsx` (no longer used) was deleted. Then, `BudgetCard-V2.jsx` was renamed to `BudgetCard.jsx`. Otherwise, unchanged.